Lab_5.ipynb

```diff
@@ -0,0 +1,978 @@
+   "id": "0ee62b71-2282-4945-94cb-3dcfad65521e",
+   "metadata": {},
+   "source": [
+    "### Compute the expected discovery sensitivity (significance of the expected signal) by normalizing each sample appropriately (see 'expected yields' below)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dec3ead2-65a5-45c6-8fe1-d9e992cd33f2",
+   "metadata": {},
+   "source": [
+    "The expected yields of our dataset indiciate that with 20,000 jets we would see 100 true signals: $ N_{Higgs} = 100 $ and $ N_{QCD} = 20,000 $. Since the signals are discrete and independent we know the background to Poisson distributed with $ \\lambda = 20,000 $ jets. Since the $ N_{Higgs} = 100 $ then this means that with $ \\lambda $ + $ N_{Higgs} $ would give us an expected Higgs Boson signal. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "id": "3568ba96-b1cc-4202-a58e-e97eadf9fec3",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The significance of a 20,100 signal is  0.7112259282313185 sigma.\n"
+     ]
+    }
+   ],
+   "source": [
+    "lam = 20000\n",
+    "signal = 20100\n",
+    "\n",
+    "probability = stats.poisson.sf(signal, lam)\n",
+    "sigma = stats.norm.isf(probability)\n",
+    "print('The significance of a 20,100 signal is ' , sigma, 'sigma.')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "30a239a9-5464-4371-a93b-3d0dc544c3f4",
+   "metadata": {},
+   "source": [
+    "This is not very significant, we would need a sigma value of 5 or more to be considered significant. We can try to find what signal would give us a 5 sigma discovery."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "2d01ef7c-f61a-42dc-bb3c-f2ed0e187e23",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "In order to find a 5 sigma discovery we would need 20711.0 jets\n"
+     ]
+    }
+   ],
+   "source": [
+    "sigma_new = 5\n",
+    "probability_new = stats.norm.sf(sigma_new)\n",
+    "signal_new = stats.poisson.isf(probability_new, lam)\n",
+    "print('In order to find a 5 sigma discovery we would need', signal_new, 'jets')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0c779e55-b170-4633-a0ca-6ddfd607a074",
+   "metadata": {},
+   "source": [
+    "Let's check if this is correct:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "id": "b2d0dac9-212e-430f-b0b7-6a7c9ce55056",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The significance of a 21,000 signal is  5.002840020760749 sigma.\n"
+     ]
+    }
+   ],
+   "source": [
+    "signal2 = 20711\n",
+    "\n",
+    "probability = stats.poisson.sf(signal2, lam)\n",
+    "sigma = stats.norm.isf(probability)\n",
+    "print('The significance of a 21,000 signal is ' , sigma, 'sigma.')"
```

Comment:
Is there a particular reason you're not using the exact signal 2 number here?

Reply:
This lab is pretty short, so I didn't have much to comment on. I appreciated the way you stacked the graphs of signal and background, and I thought it was overall well formatted. Your plan to use the background signal as a filter is v good as well.